pylint/checkers/refactoring.py

```diff
@@ -143,6 +143,10 @@ class RefactoringChecker(checkers.BaseTokenChecker):
                   'instead of checking for equality against each of the values.'
                   'This is faster and less verbose.'
                  ),
+        'R1715': ('simplify chained comparison',
```

Comment:
Let's use a more verbose message, something along the lines of `Reduce chained comparisons to a < b < c`